applications/callflow/src/module/cf_ring_group.erl

```diff
@@ -134,6 +134,11 @@ start_builder(EndpointId, Member, Call) ->
       end
      ).
 
+-spec member_active(kz_json:object()) -> boolean().
+member_active(Member) ->
+    DisableUntil = kz_json:get_value(<<"disable_until">>, Member, 0),
+    calendar:datetime_to_gregorian_seconds(calendar:universal_time()) > DisableUntil.
```

Comment:
Correct - currently only the API can be used to disable the endpoint for a specified time via the `disable_until` field. `log_in_out_ring_group` uses a binary on/off - but achieves this through the same field.